backend/internal/models/game.go

```diff
@@ -4,22 +4,28 @@ import "time"
 
 type Game struct {
 	DbModel
-	SeasonID        uint
-	Teams           []Team    `json:"teams" gorm:"many2many:games_teams;"`
-	Team1LockerRoom string    `json:"home_locker_room"`
-	Team2LockerRoom string    `json:"away_locker_room"`
-	Start           time.Time `json:"start"`
-	End             time.Time `json:"end"`
-	Venue           Venue     `json:"venue"`
-	VenueID         uint
-
-	// ManagerOnCall    User      `json:"manager_on_call"`
-	// ManagerOnCallID  uint
-	// ScoreKeeper      User `json:"score_keeper"`
-	// PrimaryReferee   User `json:"primary_referee"`
-	// SecondaryReferee User `json:"secondary_referee"`
-
-	// TODO Figure out scorekeeping and game stats, I think score is just one of the game stats
-	// HomeScore int `json:"home_score"`
-	// AwayScore int `json:"away_score"`//
+	AwayTeam            Team      `json:"away_team"`
+	AwayTeamID          uint      `json:"away_team_id"`
+	AwayTeamLockerRoom  string    `json:"away_team_locker_room"`
+	AwayTeamShotsOnGoal uint      `json:"away_team_shots_on_goal"`
+	End                 time.Time `json:"end"`
+	HomeTeam            Team      `json:"home_team"`
+	HomeTeamID          uint      `json:"home_team_id"`
+	HomeTeamLockerRoom  string    `json:"home_team_locker_room"`
+	HomeTeamShotsOnGoal uint      `json:"home_team_shots_on_goal"`
+	LeagueID            uint      `json:"league_id"`
+	League              League    `json:"league"`
+	ManagerOnCall       *User     `json:"manager_on_call"`
+	ManagerOnCallID     *uint
+	PrimaryReferee      *User `json:"primary_referee"`
+	PrimaryRefereeID    uint
+	ScoreKeeper         *User `json:"score_keeper"`
+	ScoreKeeperID       *uint
+	SecondaryReferee    *User `json:"secondary_referee"`
+	SecondaryRefereeID  *uint
+	Season              Season    `json:"season_id"`
+	SeasonID            uint      `json:"season"`
+	Start               time.Time `json:"start"`
+	Venue               Venue     `json:"venue"`
+	VenueID             uint
```

Comment:
In general, are we ready for all of these foreign key associations? As in, are the associated models ready to use?

Reply:
Did we not determine that the foreign key associations will be filled with null values until they are created, so this should not be a problem?